tests/test_board.cpp

```diff
@@ -0,0 +1,26 @@
+#include "../src/coords.hpp"
+#include "test.hpp"
+#include "../src/enums.hpp"
+#include <optional>
+#include <iostream>
+#include "../src/board.hpp"
+#include <vector>
+
+using namespace TicTacToe;
+
+
+int main () {
+
+    std::cout<<"supa";
+    Board b;
+    Coords cord = {0, 2};
+    b.changeFieldContent(cord, Cross);
+    if(b.fieldContent(cord)==Cross) std::cout<<"yay";
+    std::optional <Symbol> a = (b.fieldContent(cord));

```

Comment:
you could clean this test up before committing/pushing/opening a PR; also, it's a good practice to review your own PR just after you open it, so you avoid stuff like accidental mistakes or committing dirty dev debug code

I'd add a few sanity checks to this test and, overally, make a few changes there:
- clean up all unnecessary stuff like `cout`s and leftover object inits - tests (contrary to a popular belief) are also read very often; (btw: https://en.wikipedia.org/wiki/Ninety-ninety_rule - **this** is why we always want as most readable code on master as possible)
- also add checks for at least one empty field - before and after calling `changeFieldContent`
- use `test` func util  I wrote instead of `cout`s 
  - (in another project though, we will probably move on to using some more normalized testing methods - some public testing libraries instead of our own solutions - this is dome much more often in the industry simply because such libs are maintained by someone else, are already made and usually are well documented by the devs as well as on StackOverflow lol)

Reply:
btw you might want to check out Markdown if you dont know it yet - this way you can write *such* **fancy** `comments`
```
it's also used in many more places like Messenger and Discord
```

Reply:
oops, wrong link
here's the one
https://www.goodreads.com/quotes/835238-indeed-the-ratio-of-time-spent-reading-versus-writing-is